test/test_components.py

```diff
@@ -1441,10 +1441,8 @@ def test_video_postprocess_converts_to_playable_format(self):
             bad_vid = str(test_file_dir / "bad_video_sample.mp4")
             assert not processing_utils.video_is_playable(bad_vid)
             shutil.copy(bad_vid, tmp_not_playable_vid.name)
-            _ = gr.Video().postprocess(tmp_not_playable_vid.name)
-            # The original video gets converted to .mp4 format
-            full_path_to_output = Path(tmp_not_playable_vid.name).with_suffix(".mp4")
-            assert processing_utils.video_is_playable(str(full_path_to_output))
+            output = gr.Video().postprocess(tmp_not_playable_vid.name)
```

Comment:
Can you add `monkeypatch.setenv("GRADIO_TEMP_DIR", str(tmp_path))` here (from the deleted test)? That way the local temp dir won't get filled up with lots of temp files. Will figure out a way to do that automatically for all tests going forward in a separate PR

Reply:
Thanks, sounds good! I've done it now, but yeah ideally we set that env variable for all of the pytests at once